### PR TITLE
add new download/cloud/autopilot page

### PR DESCRIPTION
**Pull request by deadlight**
## Done

Added new /download/cloud/autopilot page

## QA

Go to `/download/cloud/autopilot` and check against the [copydoc](https://docs.google.com/document/d/1zZFBl_P0VYUSHAN7TbNLSVjDjx-zNoJFayxO8tFnCtM/edit#heading=h.4rkgldloed83)

### CODE REVIEW

File: templates/download/cloud/autopilot.html [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+{% block content %}
+
+<section class="row row-grey no-border">
+  <div class="strip-inner-wrapper">
+    <div class="eight-col">
+      <h1>Get a production cloud with OpenStack Autopilot</h1>
+    </div>
+    <div class="box box-highlight clearfix equal-height--vertical-divider">
+      <div class="equal-height--vertical-divider__item six-col">
+        <h2>Install OpenStack Autopilot on your&nbsp;hardware</h2>
```
Comment (anthonydillon) [inline templates/download/cloud/autopilot.html]: The copy docs states `OpenStack Autopilot`.

File: templates/download/cloud/autopilot.html [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        <p>Canonical&rsquo;s OpenStack Autopilot handles every aspect of your production cloud across multiple physical machines through installation, expansion and everyday operations.</p>
+      </div>
+      <div class="equal-height--vertical-divider__item six-col last-col no-margin-bottom download-button">
+        <h3>Requirements</h3>
+        <ul class="list-ticks--compact">
+          <li>A minimum of 5 machines with the following roles:
+            <ul class="list-ticks--compact">
+              <li>1 machine for the MAAS server</li>
+              <li>1 machine for the Autopilot</li>
```
Comment (anthonydillon) [inline templates/download/cloud/autopilot.html]: `1 machine for the OpenStack Autopilot`

File: templates/download/cloud/autopilot.html [hunk 3]
```diff
@@ -0,0 +1,266 @@
+{% extends "download/_base_download.html" %}
+
+{% block title %}Build OpenStack with Autopilot| Download{% endblock %}
```
Comment (anthonydillon) [inline templates/download/cloud/autopilot.html]: ` | Download` Add space before pipe

File: templates/download/cloud/autopilot.html [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+{% block extra_body_class %}download-cloud-conjure-up{% endblock %}
```
Comment (anthonydillon) [inline templates/download/cloud/autopilot.html]: This can be removed.

**Comment by deadlight** [comment]
@anthonydillon ready for re-review

**Pull request merged**